ui/app/abilities/job.js

```diff
@@ -29,4 +32,9 @@ export default class Job extends AbstractAbility {
   get policiesSupportScaling() {
     return this.namespaceIncludesCapability('scale-job');
   }
+
+  @computed('rulesForNamespace.@each.capabilities')
+  get policiesSupportDispatching() {
```

Comment:
I think we'll also need a test and some refactoring of existing tests to account for `dispatch-job` in `abilities/job-test`